2022-03-08 10:01:45,126 INFO    MainThread:24529 [wandb_setup.py:_flush():69] setting env: {'entity': 'chiquita', 'project': 'msynth-tests', 'sweep_id': 'jnm1hb79', 'root_dir': '/proj/vondrick/mia/msynth/tests', 'run_id': '4v0f5lht', 'sweep_param_path': '/proj/vondrick/mia/msynth/tests/wandb/sweep-jnm1hb79/config-4v0f5lht.yaml'}
2022-03-08 10:01:45,126 INFO    MainThread:24529 [wandb_setup.py:_flush():69] setting login settings: {}
2022-03-08 10:01:45,127 INFO    MainThread:24529 [wandb_init.py:_log_setup():336] Logging user logs to /proj/vondrick/mia/msynth/tests/wandb/run-20220308_100145-4v0f5lht/logs/debug.log
2022-03-08 10:01:45,127 INFO    MainThread:24529 [wandb_init.py:_log_setup():337] Logging internal logs to /proj/vondrick/mia/msynth/tests/wandb/run-20220308_100145-4v0f5lht/logs/debug-internal.log
2022-03-08 10:01:45,128 INFO    MainThread:24529 [wandb_init.py:init():369] calling init triggers
2022-03-08 10:01:45,128 INFO    MainThread:24529 [wandb_init.py:init():376] wandb.init called with sweep_config: {'epsilon': 0.01, 'learning_rate': 0.0001, 'num_avg': 20, 'num_notes': 16}
config: {}
2022-03-08 10:01:45,128 INFO    MainThread:24529 [wandb_init.py:init():418] starting backend
2022-03-08 10:01:45,128 INFO    MainThread:24529 [backend.py:_multiprocessing_setup():71] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-03-08 10:01:45,148 INFO    MainThread:24529 [backend.py:ensure_launched():123] starting backend process...
2022-03-08 10:01:45,161 INFO    MainThread:24529 [backend.py:ensure_launched():128] started backend process with pid: 24652
2022-03-08 10:01:45,162 INFO    MainThread:24529 [wandb_init.py:init():423] backend started and connected
2022-03-08 10:01:45,164 INFO    MainThread:24529 [wandb_run.py:_config_callback():749] config_cb None None {'epsilon': 0.01, 'learning_rate': 0.0001, 'num_avg': 20, 'num_notes': 16}
2022-03-08 10:01:45,165 INFO    MainThread:24529 [wandb_init.py:init():463] updated telemetry
2022-03-08 10:01:45,165 INFO    MainThread:24529 [wandb_init.py:init():482] communicating current version
2022-03-08 10:01:46,850 INFO    MainThread:24529 [wandb_init.py:init():487] got version response upgrade_message: "wandb version 0.12.11 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-03-08 10:01:46,850 INFO    MainThread:24529 [wandb_init.py:init():495] communicating run to backend with 30 second timeout
2022-03-08 10:01:47,029 INFO    MainThread:24529 [wandb_init.py:init():520] starting run threads in backend
2022-03-08 10:01:47,259 INFO    MainThread:24529 [wandb_run.py:_console_start():1524] atexit reg
2022-03-08 10:01:47,261 INFO    MainThread:24529 [wandb_run.py:_redirect():1371] redirect: SettingsConsole.REDIRECT
2022-03-08 10:01:47,262 INFO    MainThread:24529 [wandb_run.py:_redirect():1376] Redirecting console.
2022-03-08 10:01:47,266 INFO    MainThread:24529 [wandb_run.py:_redirect():1438] Redirects installed.
2022-03-08 10:01:47,266 INFO    MainThread:24529 [wandb_init.py:init():544] run started, returning control to user process
2022-03-08 10:01:47,269 INFO    MainThread:24529 [wandb_run.py:_atexit_cleanup():1494] got exitcode: 1
2022-03-08 10:01:47,271 INFO    MainThread:24529 [wandb_run.py:_restore():1466] restore
2022-03-08 10:01:49,386 INFO    MainThread:24529 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 867
  total_bytes: 867
}

2022-03-08 10:01:49,686 INFO    MainThread:24529 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 867
  total_bytes: 867
}

2022-03-08 10:01:50,072 INFO    MainThread:24529 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 867
  total_bytes: 7342
}

2022-03-08 10:01:50,245 INFO    MainThread:24529 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7342
  total_bytes: 7342
}

2022-03-08 10:01:50,348 INFO    MainThread:24529 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7342
  total_bytes: 7342
}

2022-03-08 10:01:50,451 INFO    MainThread:24529 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7342
  total_bytes: 7342
}

2022-03-08 10:01:50,555 INFO    MainThread:24529 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7342
  total_bytes: 7342
}

2022-03-08 10:01:50,658 INFO    MainThread:24529 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7342
  total_bytes: 7342
}

2022-03-08 10:01:50,761 INFO    MainThread:24529 [wandb_run.py:_wait_for_finish():1616] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7342
  total_bytes: 7342
}

